pkg/stdlib/connector.go

```diff
@@ -27,13 +28,20 @@ type driverConnector struct {
 }
 
 // Connect implement driver.Connector interface
-func (c driverConnector) Connect(ctx context.Context) (driver.Conn, error) {
-	cliOptions, err := ParseConfig(c.name)
-	if err != nil {
-		return nil, err
+func (c *driverConnector) Connect(ctx context.Context) (driver.Conn, error) {
+	var immuClientOption *client.Options
+	c.driver.configMutex.Lock()
+	immuClientOption = c.driver.clientOptions[c.name]
```

Comment:
immuClientOption := c.driver.clientOptions[c.name]

probably it's more compact